travvy-server/src/models/Attractions.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (sequelize, DataTypes) =>
+  sequelize.define('Attractions', {
+    attraction_id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true
+    },
+    attraction_name: DataTypes.TEXT,
+    location_id: DataTypes.INTEGER,
+    city: DataTypes.TEXT,
+    country: DataTypes.TEXT,
+    email_address: DataTypes.TEXT,
+    phone_number: DataTypes.TEXT,
+    street_address: DataTypes.TEXT,
+    opening_date: DataTypes.TEXT,
+    closing_date: DataTypes.TEXT,
+    max_number_guest: DataTypes.TEXT,
+    ticket_price: DataTypes.TEXT,
```

Comment:
(like Lucas said before) are we sure these should all be text? some seem like ints or doubles

Reply:
yup you both are right, i could change the max_number_guest to be int but the rest must stay at text

Reply:
this is because certain values in the database take forms of "integer + text" and since we dont plan on doing any further calculations with them, its better as text

Reply:
ticket price has to be text because of the currency Rushan mentioned. “$25 USD” or “£15 EURO”